### ISSUE TITLE
Typo in add_sequential_constr

### ISSUE THREAD

**Issue by fuadcan** [Gurobi/gurobi-machinelearning#312]
Small thing. The package supports only torch.nn.linear and torch.nn.ReLU but the documentation says "Decision variables used as input for logistic regression". I think you meant regression.

Also, I noticed you support logistic regressions now. Do you plan to add torch model with sigmoid activation functions later?

**Comment by pobonomo** [comment]
Thanks for reporting the typo. I'll fix it with #313 

Regarding logistic regression, we support scikit-learn's logistic regression with two classes currently.

Supporting neural networks with sigmoid activation is something we are thinking about but didn't do so far (adding the model is not very difficult but we will have to see how hard it is to solve then).

**Comment by pobonomo** [comment]
It should be fixed now https://gurobi-machinelearning.readthedocs.io/en/stable/auto_generated/gurobi_ml.torch.sequential.add_sequential_constr.html#gurobi_ml.torch.sequential.add_sequential_constr

For the sigmoid, it is something we are considering. As Gurobi is adding better and better support for nonlinear functions, we will wait a bit.
